extmod/modubinascii.c

```diff
@@ -33,7 +33,6 @@
 #include "py/binary.h"
 #include "modubinascii.h"
 
-
```

Comment:
Why would I want to get rid of the .exp files? Have I misunderstood how the test framework is supposed to work? Without them, how do we know if we're getting the right results? Or do we only care that the code runs without errors?